lib/active_set/sorting/enumerable_strategy.rb

```diff
@@ -14,7 +14,12 @@ def execute
         # http://brandon.dimcheff.com/2009/11/18/rubys-sort-vs-sort-by/
         @set.sort_by do |item|
           @attribute_instructions.map do |instruction|
-            sortable_numeric_for(instruction, item) * direction_multiplier(instruction.value)
+            value_for_comparison = sortable_numeric_for(instruction, item)
+            if value_for_comparison.nil?
+              [direction_multiplier(instruction.value), 0]
```

Comment:
I'm going to forget how these tuples effect sorting behavior. Maybe we should add a comment explaining as you did in the PR description what the behavior will be and why this is implemented like so.

Reply:
Thanks. See if the comment in https://github.com/fractaledmind/actionset/pull/27/commits/2dbdb1794d476216f53764fd10b1ac1291d8e1eb works for you.